### DISCUSSION TITLE
Is it time for zune-inflate announcement?

### DISCUSSION THREAD

**Discussion by Shnatsel** [etemesi254/zune-image#85] [General]
`zune-inflate` appears to be ready for production use. It might be a good idea to announce it before PNG and JPEG. More independent announcements usually leads to more exposure.

I've written a draft of the release announcement: https://hackmd.io/@xmxSgN6MSImJPcY4MOKfjA/ryZ__abpi

Editing is open to everyone with a link, so feel free to edit as you see fit. I recommend posting the announcement to [/r/rust](https://www.reddit.com/r/rust/).

**Comment by etemesi254** [comment]
Have to admit I'm not so good with release announcements as this is the first "complete" package I've ever done.


Also I think it would be better if you spearhead this, as you probably have a bigger voice  in the community than me, so t would probably have a greater impact.

**Comment by Shnatsel** [comment]
Alright. I've expanded the announcement following the structure you suggested.

Please let me know if you're OK with the revised text. If yes, I'll post it tomorrow.

**Comment by etemesi254** [comment]
Looks good to me.

Thanks for the fuzz testing and everything you did.

I may lurk in the Reddit thread to answer questions of need be

**Comment by Shnatsel** [comment]
Posted: https://redd.it/10zq906

**Reply by Shnatsel** [reply]
It may have accidentally been flagged as spam because I don't see it show up on the actual subreddit.

Reddit has been giving me error 500 earlier so I can't tell if this is a glitch or what.

**Reply by Shnatsel** [reply]
Well, Reddit is clearly misbehaving, so I'll post the announcement tomorrow instead.

**Reply by etemesi254** [reply]
No problem

**Comment by Shnatsel** [comment]
Posted again: https://redd.it/110hxc9

**Reply by Shnatsel** [reply]
Just so you know, Reddit reports a total of 40,000 views and a 98% upvote rate on this announcement.
